src/api/handlers/andis.js

```diff
@@ -0,0 +1,15 @@
+import Axios from 'axios';
+
+const options = {
+  method: 'GET',
+  headers: {
+    'x-api-key' : 'YsO9MGt0DwUu2PDTQ4y4TrhRtnJlYFzYWa1NW92PmvDFqcyDGLiacSNDWape'
```

Comment:
as discussed, x-api-key needs removing

Reply:
Can you replace this with `process.env.AUTHENTICATION_TOKEN` 

I'll put a PR in for a change to master that includes these env variables